share/revng/test/tests/analysis/Decompilation/x86-64/dual-case-switch.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is distributed under the MIT License. See LICENSE.md for details.
+ */
+
+#define WEAK __attribute__((weak))
+
+WEAK void report(int code) {
+}
+
+int f(int arg1) {
```

Comment:
Now I wonder why `f` is the only one not marked `WEAK`. Any specific reason?

Reply:
No specific reason except my stupidity:) Fixed this in 91cd7fca9e648fe8c5b9286dc3909ed9cf9b83b7